templates/drupal9/files/.platform.app.yaml

```diff
@@ -104,23 +120,24 @@ web:
                 '^/sites/[^/]+/settings.*?\.php$':
                     scripts: false
 
+        # The files directory has its own special configuration rules.
         '/sites/default/files':
             # Allow access to all files in the public files directory.
             allow: true
             expires: 5m
             passthru: '/index.php'
             root: 'web/sites/default/files'
 
-            # Do not execute PHP scripts.
+            # Do not execute PHP scripts from the writeable mount.
             scripts: false
 
             rules:
                 # Provide a longer TTL (2 weeks) for aggregated CSS and JS files.
                 '^/sites/default/files/(css|js)':
                     expires: 2w
 
-# The configuration of scheduled execution.
 crons:
+    # Run Drupal's cron tasks every 19 minutes.
     drupal:
-        spec: '*/20 * * * *'
+        spec: '*/19 * * * *'
```

Comment:
cicada

Reply:
They're everywhere in Chicago this year.